server/modules/user/manager/userManager.js

```diff
@@ -1,35 +1,60 @@
-const R = require('ramda')
+const db = require('../../../db/db')
 
 const UserRepository = require('../repository/userRepository')
-const AuthGroupRepository = require('../../auth/repository/authGroupRepository')
+const AuthManager = require('../../auth/manager/authManager')
+
+const User = require('../../../../common/user/user')
+
+const ActivityLog = require('../../activityLog/activityLogger')
+
+// ==== CREATE
+
+const insertUser = async (user, surveyId, email, groupId, client = db) =>
+  client.tx(async t => {
+    const newUser = await UserRepository.insertUser(surveyId, email, groupId, t)
+    await ActivityLog.log(user, surveyId, ActivityLog.type.userJoined, { email, id: User.getId(newUser) }, t)
+
```

Comment:
has here the user accepted the invitation? ActivityLog.type.userJoined should be inserted when the user accept the invitation.